Sources/TokamakStaticHTML/Views/Navigation/NavigationView.swift

```diff
@@ -16,14 +16,17 @@ import TokamakCore
 
 extension NavigationView: ViewDeferredToRenderer {
   public var deferredBody: AnyView {
-    AnyView(HTML("div", [
-      "class": "_tokamak-navigationview",
+    let proxy = _NavigationViewProxy(self)
+    print(proxy.toolbar)
+    return AnyView(HTML("div", [
+      "class": "_tokamak-navigationview \(proxy.toolbar != nil ? "_tokamak-navigationview-with-toolbar" : "")",
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakStaticHTML/Views/Navigation/NavigationView.swift;
  Line: 22;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 111 characters (`line_length`)